x/stake/keeper/test_common.go

```diff
@@ -202,5 +202,22 @@ func createTestPubKeys(numPubKeys int) []crypto.PubKey {
 // does a certain by-power index record exist
 func ValidatorByPowerIndexExists(ctx sdk.Context, keeper Keeper, power []byte) bool {
 	store := ctx.KVStore(keeper.storeKey)
-	return store.Get(power) != nil
+	return store.Has(power)
+}
+
+func updateValidator(keeper Keeper, ctx sdk.Context, validator types.Validator) types.Validator {
```

Comment:
Can this be renamed to `testingUpdateValidator`, so its not mistaken for a function that should be used in the code

Reply:
actually noticed this too and thought the same thing.

++

Reply:
So renamed.